index.js

```diff
@@ -47,7 +47,7 @@ module.exports = (options = {}) => {
 		fancyLog(title + what + ' ' + sizeStrings.join(chalk.magenta(', ')));
 	}
 
-	return through.obj((file, encoding, callback) => {
+	return through.obj(async (file, encoding, callback) => {
```

Comment:
I intentionally did not make the whole callback function async. If anything throws, it will cause an unhandled error. Use the async-IIFE like before.

Reply:
I should have guessed...